rjfoley/style.css

```diff
@@ -0,0 +1,23 @@
+/* Import fonts */
+
+/* Re-styling Basics i.e.: body h1 h2 h3 ... ; */
+
+/* Snippet Title */
+
+.only-text {
+	font-size:24px;
+	line-height: 1.2em;
+	font-family: "Helvetica Neue";
+	max-width: 720px
+	padding: 5%;
+	color: turquoise;
+	margin:auto;
+	display: inline-block;
+	float: left;
+}
```

Comment:
Also nit picky, but I might change the order of your CSS properties so that things that effect the box model (display, float, margin, padding and max-width) come first.

Reply:
Sure thing. A lot of ID folks have preferences for either alphabetic or ordered principles https://github.com/bu-ist/coding-standards/blob/develop/css-and-sass/README.md#declaration-order